HOW_TO_RELEASE.md

```diff
@@ -53,11 +71,19 @@ Time required: about an hour.
 14. Issue the release announcement! For bug fix releases, I usually only email
     xarray@googlegroups.com. For major/feature releases, I will email a broader
     list (no more than once every 3-6 months):
-      pydata@googlegroups.com, xarray@googlegroups.com,
-      numpy-discussion@scipy.org, scipy-user@scipy.org,
-      pyaos@lists.johnny-lin.com
+      - pydata@googlegroups.com
+      - xarray@googlegroups.com
+      - numpy-discussion@scipy.org
+      - scipy-user@scipy.org
+      - pyaos@lists.johnny-lin.com
+
     Google search will turn up examples of prior release announcements (look for
     "ANN xarray").
+    You can get a list of contributors with script (NB: this has limited testing),
+    replacing `v0.X.Y` with the _previous_ release
+    ```
+    git log v0.X.Y.. --format="%aN" --reverse | perl -e 'my %dedupe; while (<STDIN>) { print unless $dedupe{$_}++}' | sort
```

Comment:
I'm not sure if that option is always available (could be a GNU extension), but this could be
```suggestion
git log v0.X.Y.. --format="%aN" --reverse | sort -u
```